### DISCUSSION TITLE
How can i validate my email for an account of vercel.com

### DISCUSSION THREAD

**Discussion by Magic-ship** [vercel/community#750] [Help] [answered]
Hello everyone. 
My name is Magic Johnson.
Here are two issues what i need your website.
1. My email isn't available validate.
2. I can't login your website.

**Comment by amyegan** [comment, marked as answer]
Hi @Magic-ca. Thanks for reaching out! It looks like you already have a ticket in our system and someone from the team is working with you. You should have an email from them in your inbox. Since we want to keep your private information safely out of this public discussion, please follow up with that ticket for assistance.
